server/services/gameLogic.js

```diff
@@ -53,7 +53,7 @@ exports.getCurrentPlayer = () => currentPlayer;
 exports.getCardsByUsername = (username) => players.find(player => player.username === username).cards.filter(card => card.played === false);
 
 /* Returns true if the player is able to join the game, false otherwise */
-exports.canJoinGame = (username) => (status === statusTypes.NOT_STARTED && !players.some(player => player.username === username));
+exports.canJoinGame = (username) => ((status === statusTypes.NOT_STARTED || status === statusTypes.GAME_OVER) && !players.some(player => player.username === username));
```

Comment:
When I added the `statusTypes.GAME_OVER` I was thinking it would be used for the stage in the game when scores are displayed etc so we don't want people joining the game at that point. Then when someone then gets a new game ready the status would go back to `NOT_STARTED` and people could join again

Reply:
Right, I see... I'll remove that, it was mainly for testing if the game finished when all the players had voted